Objects/setobject.c

```diff
@@ -1694,9 +1709,10 @@ set_symmetric_difference(PySetObject *so, PyObject *other)
 }
 
 PyDoc_STRVAR(symmetric_difference_doc,
-"Return the symmetric difference of two sets as a new set.\n\
+"symmetric_difference($self, other, /)\n\
+--\n\
 \n\
-(i.e. all elements that are in exactly one of the sets.)");
+Return a new set with elements in either the set or other but not both.");
```

Comment:
BTW, for other set methods it seems there are signatures.  But...

An example:
```
>>> help(set.add)
Help on method_descriptor:

add(self, object, /)
    Add an element to a set.

    This has no effect if the element is already present.
```
c.f. the sphinx docs:
```
Add element elem to the set.
```

Reply:
Signatures for methods without arguments and with a single positional argument are now autogenerated.

Reply:
Anyway, this looks like a bug.

Reply:
What exactly?

Reply:
I was hoping is obvious.  Instead of mystic name ``object``, I would expect something like this
```pycon
>>> inspect.signature(set.add)
<Signature (self, element, /)>
>>> help(set.add)
Help on method_descriptor:

add(self, element, /)
    Add an element to a set.

    This has no effect if the element is already present.
```
I realize, parameter name here is not a part of API, but I think it should be meaningful, if possible.

Reply:
It could be improved, but it is not a bug. You can add explicit signatures for these methods, but for me it has low priority. I worked on support of multi-signatures, and this PR is a side effect, improvements that do not need multi-signatures (with multi-signatures and autogenerated signatures almost 100% of builtin functions and classes will have signatures).

Reply:
I think 'object', an instance of the base object class, is as correct as 'element', which is a set-theory specific synonym.  We might say that 'a list has items', but `item` is a synonym for `object`.  Here also, I appreciate the technically correct `list.append(self, object, /)`.  I think the consistency of the auto-generated signatures is good.

Reply:
One could argue that it should be `set.add(self, hashable)`, as not any object can be added, whereas any object can be appended.  The math term 'element' does not imply the restriction any more than 'object'.